### ISSUE TITLE
All progress and settings in app get reset completely unprompted

### ISSUE THREAD

**Issue by a-kbd** [davidhealey/waistline#735]
This happens on Samsung galaxy note 9, latest version of waistline installed from f-droid

I don't know how to reproduce it or how to provide you with more information but every 1-2 days, when opening the app, every setting and any entry in the app are reset and gone

It's such a good app otherwise and seemingly the only good open source alternative on the phone

Please let me know how to provide you with more information/logs about the bug

**Comment by davidhealey** [comment]
Someone reported that when the webview memory gets full it starts purging stuff. Perhaps that is what is happening in your case.

**Comment by a-kbd** [comment]
How can I tell?

Since I've been playing around with "uninstalling", i.e. disabling the chrome app on the phone perhaps it's related to that then?

**Comment by davidhealey** [comment]
Yes playing around with the chrome app could be the cause. I'm not sure how to tell. Here's the original issue that reported this - https://github.com/davidhealey/waistline/issues/728

**Comment by davidhealey** [comment]
It's not going to happen unless there is some drop in replacement or somebody else takes on the task because I don't have the time to do it
